### PR TITLE
Modification in Note Section

### PR DESCRIPTION
**Pull request by joinimran**
As mentioned in other GCC or DoD documentation, it states that GCC or DoD is only available using volume licensing.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9213

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-10-subscription-activation.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ The following figure illustrates how deploying Windows 10 has evolved with each
 > The following requirements do not apply to general Windows 10 activation on Azure. Azure activation requires a connection to Azure KMS only, and supports workgroup, Hybrid, and Azure AD-joined VMs. In most scenarios, activation of Azure VMs happens automatically. For more information, see [Understanding Azure KMS endpoints for Windows product activation of Azure Virtual Machines](https://docs.microsoft.com/azure/virtual-machines/troubleshooting/troubleshoot-activation-problems#understanding-azure-kms-endpoints-for-windows-product-activation-of-azure-virtual-machines).
 
 > [!NOTE]
-> Currently, Subscription Activation is only available on commercial tenants and is not currently available on US GCC or GCC High tenants.
+> Currently, Subscription Activation is only available on commercial tenants and is not currently available on US GCC,GCC High or, DoD tenants.
```
Comment (illfated) [inline windows/deployment/windows-10-subscription-activation.md:94]: Grammar, spacing, and punctuation improvement suggestion:
```suggestion
> Currently, Subscription Activation is only available on commercial tenants and is currently not available on US GCC, GCC High, or DoD tenants.
```

Comment (illfated) [reply windows/deployment/windows-10-subscription-activation.md:94]: I found myself pondering if the word "not" should be highlighted with **bold** text, but that kind of formatting is normally not used in MS Docs codestyle.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
Hi @greg-lindsay , PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**